src/components/common/NavMenu.js

```diff
@@ -19,7 +19,8 @@ const NavMenu = props => {
   const context = useContext(UserContext);
   const adminContext = useContext(AdminContext);
 
-  const num = adminContext.memberReactions.Length;
+  const num = adminContext.memberReactions;
+  console.log(adminContext.memberReactions);
```

Comment:
No need for console.logs please remove from main.

Reply:
Nice catch! On it!